src/containers/settings/index.tsx

```diff
@@ -45,163 +55,115 @@ const UserInformationText = styled(Paragraph)`
-              <FormTitle>Profile</FormTitle>
-              <UserInformationText>{userFullName}</UserInformationText>
-              <Form name="change-username">
-                <Form.Item {...formLayout} name="username">
-                  <Input placeholder="Username" />
-                </Form.Item>
-              </Form>
-              <FormTitle>Change Email</FormTitle>
-              <UserInformationText>{userEmail}</UserInformationText>
-              <Form
-                {...formLayout}
-                name="change-email"
-                onFinish={onFinishChangeEmail}
-              >
-                <Form.Item
-                  {...formLayout}
-                  name="new-email"
-                  rules={[
-                    {
-                      required: true,
-                      message: 'Please input a new email address!',
-                    },
-                    {
-                      pattern: /^\S+@\S+\.\S{2,}$/,
-                      message: 'Not a valid email address',
-                    },
-                  ]}
-                >
-                  <Input placeholder="New Address" />
-                </Form.Item>
-                <Form.Item
-                  {...formLayout}
-                  name="password"
-                  rules={[
-                    {
-                      required: true,
-                      message: 'Please input your current password!',
-                    },
-                  ]}
-                >
-                  <Input placeholder="Password" />
-                </Form.Item>
-                <Form.Item>
-                  <SubmitButton type="primary" htmlType="submit">
-                    Save
-                  </SubmitButton>
-                </Form.Item>
-              </Form>
-            </Col>
-
-            <Col span={cSpan}>
-              <FormTitle>Change Password</FormTitle>
-              <Form
-                {...formLayout}
-                name="change-password"
-                onFinish={onFinishChangePassword}
-              >
-                <Form.Item
-                  name="currentPassword"
-                  rules={[
-                    {
-                      required: true,
-                      message: 'Please input your current password!',
-                    },
-                  ]}
-                >
-                  <Input placeholder="Current Password" />
-                </Form.Item>
-                <Form.Item
-                  name="newPassword"
-                  help={passwordRules}
-                  rules={[
-                    {
-                      required: true,
-                      message: 'Please input your new password!',
-                    },
-                    {
-                      min: 8,
-                      message: passwordRules,
-                    },
-                  ]}
+      <Helmet>
+        <title>Settings</title>
+        <meta
+          name="Settings"
```

Comment:
⛏️ This meta tag doesnt do anything, there is no meta tag called "Settings", I think this is supposed to be a "description" meta tag